headline.css

```diff
@@ -77,3 +69,109 @@ ul {
   background-color: #fff;
   color: black;
 }
+
+a {
+  text-decoration: none;
+}
+
+@media only screen and (max-width: 769px) {
+  .desktop__header {
+    display: none;
+  }
+
+  #desktop__geometry {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    min-height: 550px;
+  }
+
+  .hello__dots {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 769px) {
+  .toolbar {
+    box-shadow: 0 0.5px 0 #dddfe6;
+    display: grid;
+    grid-template-columns: repeat(3, 33%);
+    height: 10vh;
+    align-content: center;
+  }
+
+  .mobile__header {
+    display: none;
+  }
+
+  .desktop__header {
+    display: flex;
+    justify-content: space-between;
+    grid-column: 2/3;
+  }
+
+  .header__navication {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.5vw;
+    line-height: 24px;
+    color: #1a2236;
```

Comment:
[OPTIONAL] great job with the project, :clap: , really well done, Just please make your header links colors similar to the figma template design right now your color property is not implemented since they are inside the `li` tag  `<li class="header__navication"><a href="#headline">Hello</a></li>` you can use `.header__navication > a { color: "value" }` 
your header design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200113441-e0f01440-b247-4638-a318-1c853d40298c.png)
figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200113455-dd53ad9e-1780-4cae-a36b-691079987feb.png)